.github/workflows/test-wheel-linux.yml

```diff
@@ -286,6 +292,11 @@ jobs:
       - name: Set up compute-sanitizer
         run: setup-sanitizer
 
+      - name: Run cuda.pathfinder tests with see_what_works
```

Comment:
Not a blocker for getting this in, but in theory since we're just testing the viability of finding and loading libraries, we shouldn't need to run these tests with a GPU where we may need to split these tests into a separate workflow.

I think the only tricky part would be handling the driver, `libcuda`, where I'm not sure how easy it is to install and load without a GPU.

Reply:
That sounds good.

I think it'll be useful to keep the `see_what_works` testing here, for very inexpensive extra test coverage, but probably testing can and should indeed be run on machines without a GPU.

Currently `libcuda` isn't covered by the pathfinder.

Reply:
That sounds good.

I think it'll be useful to keep the `see_what_works` testing here, for very inexpensive extra test coverage, but probably testing can and should indeed be run on machines without a GPU.

Currently `libcuda` isn't covered by the pathfinder.